test/ruby/test_compile_prism.rb

```diff
@@ -562,6 +562,59 @@ def test_BreakNode
       assert_prism_eval("while true; break 1; end")
     end
 
+    def test_EnsureNode
+      # prints 1 2
+      assert_prism_eval(<<~ENS)
+        begin
+          puts 1
+        ensure
+          puts 2
+        end
+      ENS
+
+      # prints 1 3 4 2
+      assert_prism_eval(<<~ENS)
+        begin
+          puts 1
+          begin
+            puts 3
+          ensure
+            puts 4
+          end
+        ensure
+          puts 2
+        end
+      ENS
+
+      # prints 1 4 2
+      assert_prism_eval(<<~ENS)
+        begin
+          puts 1
+          begin
+            raise "bang"
+            puts 3
+          ensure
+            puts 4
+          end
+        ensure
+          puts 2
+        end
+      ENS
+
+      # prints 1 2 hi
+      assert_prism_eval(<<~ENS)
+        begin
+          puts 1
+        ensure
+          puts 2
+        end
+        puts "hi"
+      ENS
+
+    # popped
```

Comment:
I don't think this case is necessary because of [this](https://github.com/ruby/ruby/blob/master/test/ruby/test_compile_prism.rb#L1005-L1006)